tools/perspective-scripts/test_js.mjs

```diff
@@ -104,7 +104,9 @@ if (process.env.PACKAGE) {
         process.env.PACKAGE.indexOf("perspective-python") >= 0 &&
         process.env.PACKAGE.indexOf("!perspective-python") === -1
     ) {
-        sh`pnpm run --recursive --filter @finos/perspective-python test`.runSync();
+        // Support `pnpm test -- --my_cool --test_arguments`
```

Comment:
new feature -- I wanted to be able to thread pytest arguments down through a top-level `pnpm run test`